lang/pt.js

```diff
@@ -1,25 +1,25 @@
 /* eslint quotes: 0 */
 
 export default {
-	"components.alert.close": "Fechar Alerta",
+	"components.alert.close": "Fechar alerta",
 	"components.calendar.notSelected": "Não selecionado.",
 	"components.calendar.selected": "Selecionado.",
 	"components.calendar.show": "Mostrar {month}",
 	"components.dialog.close": "Fechar esta caixa de diálogo",
-	"components.input-date.clear": "Desmarcar",
-	"components.input-date.openInstructions": "",
-	"components.input-date.setToToday": "",
+	"components.input-date.clear": "Limpar",
+	"components.input-date.openInstructions": "Adote o formato de data {format}. Use a tecla de seta para baixo ou pressione Enter para acessar o minicalendário.",
+	"components.input-date.setToToday": "Definir como Hoje",
 	"components.input-date-time.date": "Data",
-	"components.input-date-time.time": "Tempo",
```

Comment:
"Time" -> "Hour" 👎

Reply:
I think we might actually want "Horas" here [based on this](https://streetsmartbrazil.com/ask-say-time-portuguese-need-know/). Anyone speak Portuguese?

Reply:
Actually Google tasks uses "Definir hora" (define the time), so let's go with the new one.